account_move_name_sequence/tests/test_account_move_name_seq.py

```diff
@@ -309,7 +309,6 @@ def test_remove_invoice_error_secuence_standard(self):
                 ],
             }
         )
-        in_refund_invoice._compute_split_sequence()
```

Comment:
Why this is needed?

Reply:
When I wrote the test, I believed it was necessary. Checking the error, I see that it is not necessary.